docs/topics/multiplatform/multiplatform-discover-project.md

```diff
@@ -141,7 +141,7 @@ In a multiplatform project with native and JS targets, the following code in `co
 ```kotlin
 // commonMain/kotlin/common.kt
 // Doesn't compile in common code
-fun common() {
+fun greeting() {
```

Comment:
please see line 14 - line 25
it is `fun greeting()` in the common code

and line 149 - line 160
it is `fun jvmGreeting()` in jvm code

so I think, it should use the same `fun greeting()` here

about the file name, I think it is not so important, since Kotlin allow any file name.

in line 142, it is `commonMain/kotlin/common.kt`
in line 155, it is `jvmMain/kotlin/jvm.kt`
I think these file names are totally OK, to use different source file name for each platform

Reply:
fair point - thank you for sharing your thoughts - I agree with the change then :)